be/src/exec/pipeline/sink/table_function_table_sink_operator.cpp

```diff
@@ -224,7 +224,7 @@ Status TableFunctionTableSinkOperatorFactory::prepare(RuntimeState* state) {
         }
         _parquet_file_schema = result.ValueOrDie();
     } else {
-        return Status::InternalError("unsupported file format" + _file_format);
+        return Status::InternalError("unsupported file format: " + _file_format);
```

Comment:
I got you. The check at BE is case-sensitive. Can you modify the 219 line to make it case-insensitive?

Reply:
maybe we can use `boost::iequals`

Reply:
ok